tests/conftest.py

```diff
@@ -33,6 +33,17 @@ def backend2(requests_mock) -> str:
     return domain
 
 
+# as "lib_requests_mock": make a distinction with the pytest fixture that has the same name 
+import requests_mock as lib_requests_mock
```

Comment:
True, the type hint is in this case really only an nice to have. We can leave out this import to simplify, if we even keep this helper function, because, as you said, it would be better to leave out the test coverage for API version 0.4.0.

Reply:
Fixed: I have removed set_backend_to_api_version entirely.